ember-engines-router-service/package.json

```diff
@@ -59,9 +59,6 @@
   "peerDependencies": {
     "ember-source": "^3.28.0 || ^4.0.0"
   },
-  "engines": {
```

Comment:
Why are we removing these lines? Is there any documentation that you can link here?

Reply:
@villander this was discussed in https://github.com/embroider-build/addon-blueprint/issues/15 and latest v2 addon blueprint does not provide `engines.node`

Reply:
Yes, but we still need to support V1 addons

Reply:
@villander I don't get your concern.

This is addon is v2 addon, meaning it does not have any Node.js runtime.
v1 addons do require runtime hence any v1 should properly declare Node.js versions it supports.